pkg/rtc/mediatrack.go

```diff
@@ -56,6 +56,7 @@ type MediaTrack struct {
 }
 
 type MediaTrackParams struct {
+	BaseTime              time.Time
```

Comment:
did we end up making a global for this?

Reply:
Not yet. Haven't gone back to look at it. Been lagging :-(